Makefile

```diff
@@ -74,13 +84,13 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	debian/bin/gencontrol.py
 
 	# generate linux build file for amd64_none_amd64
-	# fakeroot make -f debian/rules.gen setup_armhf_none_armmp
-	# fakeroot make -f debian/rules.gen setup_arm64_none
-	fakeroot make -f debian/rules.gen setup_amd64_none_amd64
+	$(call dpkg-architecture-armhf) fakeroot make -f debian/rules.gen setup_armhf_none_armmp
```

Comment:
The one mentioned in this comment also does the job
https://github.com/Azure/sonic-linux-kernel/pull/121#issuecomment-601948518